django/conf/locale/ar_DZ/formats.py

```diff
@@ -2,20 +2,28 @@
 #
 # The *_FORMAT strings use the Django date format syntax,
 # see https://docs.djangoproject.com/en/dev/ref/templates/builtins/#date
-DATE_FORMAT = 'j F Y'  # 04 نوفمبر 2020
-TIME_FORMAT = 'g:i A'
-# DATETIME_FORMAT =
+DATE_FORMAT = 'j F Y'  
+TIME_FORMAT = 'H:i'
+DATETIME_FORMAT = 'j F Y H:i'
 YEAR_MONTH_FORMAT = 'F Y'
 MONTH_DAY_FORMAT = 'j F'
-SHORT_DATE_FORMAT = 'd‏/m‏/Y'
-# SHORT_DATETIME_FORMAT =
+SHORT_DATE_FORMAT = 'j N Y'
+SHORT_DATETIME_FORMAT = 'j N Y H:i'
```

Comment:
We [discussed](https://github.com/django/django/pull/12422#issuecomment-585463877) in the original PR that abbreviations of months don't exist in Algerian Arabic and we shouldn't use `N` in formats. Maybe we can change `N` -> `F` :thinking:

Reply:
Hi felixxm,

Yes I confirm this change.

and Yes we did discuss this in the last PR about short date '%y/%m/%d' and YEEES there's no short date in Algerian date, but I discussed with a coleague and she said that It would be much pleasante to have DATE_FORMAT = 'j F Y' as a fallback in place of a SHORT_DATE_FORMAT = 'd‏/m‏/Y' when a SHORT_DATE_FORMAT call occur.

You right. and Yes Sorry the 'N' should be 'F', becasue of the fall back. so It means that whetever the short month is, it has to be replace with a long version of month.
Like : "Dec" ---> "December"
Arabic : only "ديسمبر"

Specially that arabic months name are not that long than latin ;)

thank you.

Reply:
Thanks, updated.